### PR TITLE
Update cluster-autoscaler readme

### PR DESCRIPTION
**Pull request by justinsb**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1715)

### CODE REVIEW

**Comment by justinsb** [comment]
Some small tweaks.  You OK with this @ese?

**Comment by ese** [comment]
LGTM

File: addons/cluster-autoscaler/README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 # Cluster Autoscaler Addon
 
+Note that you likely want to change `AWS_REGION` and `GROUP_NAME`, and probably `MIN_NODES` and `MAX_NODES`
```
Comment (geojaz) [inline addons/cluster-autoscaler/README.md]: Maybe something like: 

- Configure the following environment variables as appropriate for your deployment.

**Comment by yissachar** [comment]
Merging because this has been dangling. Let's add more docs improvements in a new PR.

**Pull request merged**